docs/agent/changes.md

```diff
@@ -99,7 +126,7 @@ The agent has a new set of command-line options:
 | version         | Print the version info |
 
 #### Gui
-* The browser session must be restarted each time the Agent service is restarted.  This will be fixed in an upcoming beta.
+* The browser session must be restarted each time the Agent service is restarted. This will be fixed in an upcoming release.
```

Comment:
is this still true @derekwbrown?

Reply:
This is fixed

https://github.com/DataDog/datadog-agent/pull/1180
https://github.com/DataDog/datadog-agent/pull/915